### DISCUSSION TITLE
I forgot to cancel my GitHub Copilot subscription and was overcharged. How can I request a refund?

### DISCUSSION THREAD

**Discussion by 15010208518** [community/community#70387] [New to GitHub]
A few months ago, I started a trial of GitHub Copilot but forgot to cancel the subscription. As a result, I've been charged for the past few months even though I haven't actually used Copilot. How can I request a refund for these extra charges? Thank you.

**Comment by TankEngine-ish** [comment]
hey,
check out this thread - https://github.com/orgs/community/discussions/45645 
Hope it helps.
